docs/src/index.md

```diff
@@ -22,14 +22,14 @@ Their arguments must be:
   then a valid formula is `@formula(Y ~ X1 + X2)`
 - `data`: a table in the Tables.jl definition, e.g. a data frame;
   rows with `missing` values are ignored
-- `X` a matrix holding values of the independent variable(s) in columns
```

Comment:
so maybe also add features/target as this is a common term tuple in ML?

Reply:
Thanks for the great suggestion, done 👍